### PR TITLE
dev-python/pytest-forked: Version bump to 1.0.2

### PR DESCRIPTION
**Pull request by PPed72**
Closes: https://bugs.gentoo.org/679670

Signed-off-by: Paolo Pedroni <paolo.pedroni@iol.it>

### CODE REVIEW

**Comment by sbraz** [comment]
Please squash the newest commit into the first one with `git rebase`, then force-push to the branch.

**Comment by PPed72** [comment]
> Please squash the newest commit into the first one with `git rebase`, then force-push to the branch.

Done. Thanks.

File: dev-python/pytest-forked/pytest-forked-1.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-fbsd"
+IUSE="test"
+
+RDEPEND="
+	>=dev-python/pytest-2.6.0[${PYTHON_USEDEP}]"
+
+DEPEND="
+	${RDEPEND}
```
Comment (sbraz) [inline dev-python/pytest-forked/pytest-forked-1.0.2.ebuild]: You probably need `test? ( ${RDEPEND} )`

Comment (PPed72) [reply dev-python/pytest-forked/pytest-forked-1.0.2.ebuild]: Done.

**Pull request closed without merging**